src/main/java/de/tum/in/www1/artemis/domain/participation/StudentParticipation.java

```diff
@@ -56,6 +56,11 @@ public Participant getParticipant() {
         return Optional.ofNullable((Participant) student).orElse(team);
     }
 
+    @Override
+    public String getType() {
+        return "student";
```

Comment:
When I think about it, why isn't this an enum?

Reply:
(I know that the `String getType` method is not introduced in this PR)

Reply:
It's more a type-identifier than a type. The type itself is (at least in java) determined by the subclass. However, when sending the DATA as json to the client, this identifier gets added to distinguish the subclasses. Since there is no real use for this in the java-code, I would not introduce an enum here.